control-plane/agents/src/common/errors.rs

```diff
@@ -298,6 +298,8 @@ pub enum SvcError {
     },
     #[snafu(display("Snapshots for multi-replica volumes are not allowed"))]
     NReplSnapshotNotAllowed {},
+    #[snafu(display("Multi-replica volume is not allowed from snapshot"))]
```

Comment:
Snapshot belongs

Reply:
Are you suggesting the belongs to be added at the end?

Reply:
> Are you suggesting the belongs to be added at the end?

Just suggesting a modified text.

Reply:
Sounds good. Done. Thanks

Reply:
Just wondering if the message should be generic, something like "Not enough sources to create the volume replicas".
We can probably take this up later.

Reply:
Sure.